template/fullstack/authentication/client/src/App.js

```diff
@@ -0,0 +1,34 @@
+import "./App.css";
+import { Routes, Route } from "react-router-dom";
+
+import HomePage from "./pages/HomePage/HomePage";
+import ProfilePage from "./pages/ProfilePage/ProfilePage";
+import SignupPage from "./pages/SignupPage/SignupPage";
+import LoginPage from "./pages/LoginPage/LoginPage";
+
+import Navbar from "./components/Navbar/Navbar";
+import IsPrivate from "./components/IsPrivate/IsPrivate";
+import IsAnon from "./components/IsAnon/IsAnon";
+
+function App() {
+  return (
+    <div className="App">
+      <Navbar />
+
+      <Routes>      
+        <Route path="/" element={<HomePage />} />
+
+        <Route
+          path="/profile"
+          element={ <IsPrivate> <ProfilePage /> </IsPrivate> } 
+        />
+        
+        <Route path="/signup" element={<IsAnon> <SignupPage /> </IsAnon>} />
+        <Route path="/login" element={<IsAnon> <LoginPage /> </IsAnon>} />
+
+      </Routes>
+    </div>
+  );
+}
+
+export default App;
```

Comment:
Should we rename these `App.js` into `App.jsx`?

Reply:
Just noticed that we have the components as `.js`. We might want to change the extensions to `.jsx` :v: